src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -706,4 +714,13 @@ private List<RESTGekoppeldeZaakEnkelvoudigInformatieObject> listGekoppeldeZaakIn
         ));
         return enkelvoudigInformatieobjectList;
     }
+
+    private void waitForOpenZaak() {
+        try {
+            Thread.sleep(500);
```

Comment:
Does blocking the current thread for 500ms not have other unintended consequences? Maybe it's more or less ok because it is in the context of a REST service so I guess the rest of the ZAC backend is this responsive?